Dockerfile-default-gpu

```diff
@@ -32,16 +32,16 @@ RUN if [ -n "${DET_BUILD_NCCL}" ]; then /tmp/det_dockerfile_scripts/build_nccl.s
 ARG HOROVOD_NCCL_LINK=${DET_BUILD_NCCL:+STATIC}
 ARG HOROVOD_NCCL_HOME=${DET_BUILD_NCCL:+/tmp/det_nccl/build}
 
-ARG HOROVOD_PIP=horovod
```

Comment:
not 100% sure but won't you be installing package "0" if no HOROVOD_PIP argument is provided? It seems to be benign but weird.

Reply:
i think you're right, i changed it to ""